server/events/models/models.go

```diff
@@ -131,6 +131,11 @@ func NewRepo(vcsHostType VCSHostType, repoFullName string, cloneURL string, vcsU
 		return Repo{}, fmt.Errorf("invalid repo format %q, repo %q should not contain any /'s", repoFullName, owner)
 	}
 
+	azuredevopsVcsToken := ""
+	if vcsHostType == AzureDevops {
+		azuredevopsVcsToken = vcsToken
```

Comment:
Where is `vcsToken` assigned?

Reply:
https://github.com/dandcg/atlantis/blob/4ec7a87aba015cedb4b1b48dfde772dae089ef71/server/events/models/models.go#L134-L150